python/tabby/tabby-data-pipeline/predict.py

```diff
@@ -0,0 +1,201 @@
+from pathlib import Path
+
+import modal
+from modal import Image, Mount, Secret, Stub, asgi_app, gpu, method
+import os
+
+
+import asyncio
+
+GPU_CONFIG = gpu.A10G()
+
+MODEL_ID = os.popen("cat /tmp/tabby_model_id").read().strip()
+LAUNCH_FLAGS = ["serve", "--model", MODEL_ID, "--port", "8000", "--device", "cuda"]
+
+
+def download_model():
+    import subprocess
+    import os
+    MODEL_ID = os.popen("cat /tmp/tabby_model_id").read().strip()
+    print(f'MODEL_ID={MODEL_ID}')
+    subprocess.run(
+        [
+            "/opt/tabby/bin/tabby",
+            "download",
+            "--model",
+            MODEL_ID,
+        ]
+    )
+
+
+image = (
+    Image.from_registry(
+        "tabbyml/tabby:0.5.5",
+        add_python="3.11",
+    )
+    .dockerfile_commands("ENTRYPOINT []")
+    .pip_install(
+        "git+https://github.com/TabbyML/tabby.git#egg=tabby-python-client&subdirectory=experimental/eval/tabby-python-client",
+        "pandas"
+    )
+    .copy_local_file(local_path="/tmp/tabby_model_id", remote_path="/tmp/tabby_model_id")
+    .run_function(download_model)
```

Comment:
use https://modal.com/docs/reference/modal.Image#env to pass `MODEL_ID` as environment variable, and in `download_model` you can access model id as os.environ.get("MODEL_ID")

Reply:
`.env("MODEL_ID", os.environ.get("MODEL_ID"))`